homeassistant/components/binary_sensor/hikvision.py

```diff
@@ -155,8 +158,6 @@ class HikvisionBinarySensor(BinarySensorDevice):
 
     def __init__(self, hass, sensor, cam, delay):
         """Initialize the binary_sensor."""
-        from pydispatch import dispatcher
-
         self._hass = hass
```

Comment:
Is not needed. You can use `self.hass` that will set by core after `add_devices` is done.